src/main/scala/it/pagopa/pdnd/interop/uservice/partymanagement/model/persistence/State.scala

```diff
@@ -15,7 +15,7 @@ final case class State(
   private val logger = LoggerFactory.getLogger(this.getClass)
 
   def addParty(party: Party): State = {
-    logger.error(s"Writing party ${party.id.toString} to state")
+    logger.info(s"Writing party ${party.id.toString} to state")
```

Comment:
is this `info` instead of `debug` on purpose?

Reply:
typo